crates/iroha/tests/upgrade.rs

```diff
@@ -407,19 +407,11 @@ fn define_custom_parameter() -> Result<()> {
     Ok(())
 }
 
-fn upgrade_executor(client: &Client, executor: impl AsRef<str>) -> Result<()> {
-    let upgrade_executor = Upgrade::new(Executor::new(load_sample_wasm(executor)));
-    let profile = load_wasm_build_profile();
-
-    if !profile.is_optimized() {
-        client.submit_all_blocking::<InstructionBox>([InstructionBox::SetParameter(
-            SetParameter::new(Parameter::Executor(SmartContractParameter::Fuel(
-                std::num::NonZeroU64::new(90_000_000_u64).expect("Fuel must be positive."),
-            ))),
-        )])?;
+pub(crate) fn upgrade_executor(client: &Client, executor: impl AsRef<str>) -> Result<()> {
+    // client.submit_all_blocking(upgrade_executor_isi(executor))?;
+    for isi in upgrade_executor_isi(executor) {
+        client.submit_blocking(isi)?;
```

Comment:
I don't know for now why in the `submit_all` case, the executor complains about a lack of fuel.